src/xeus_sqlite_interpreter.cpp

```diff
@@ -22,98 +23,222 @@
 
 namespace xeus_sqlite
 {
-    void interpreter::configure_impl()
+
+std::vector<std::string> interpreter::tokenizer(const std::string& code)
+{
+    //TODO: this needs to be more rubust. I need to sanitize the string better
+    //removing spaces and new lines from the begginning of the string, cause
+    //those are valid inputs in a jupyter notebook, everything else can fail
+    std::stringstream input(code);
+    std::string segment;
+    std::vector<std::string> tokenized_str;
+    std::string is_magic(1, input.str()[0]);
+    tokenized_str.push_back(is_magic);
+
+    while(std::getline(input, segment, ' '))
+    {
+        tokenized_str.push_back(segment);
+    }
+
+
+    return tokenized_str;
+}
+
+bool interpreter::is_magic(std::vector<std::string>& tokenized_code)
```

Comment:
can be a `const std::vector`

Reply:
Can't be a `const` because I'm actually modifying it here to make it easier to handle later. But I could make it `const` in some other occasions, I'm doing it in other occasions. =)